meinberlin/apps/plans/views.py

```diff
@@ -77,28 +80,43 @@ def districts(self):
             return []
 
     def get_organisations(self):
-        organisations = Organisation.objects.values_list("name", flat=True).order_by(
-            "name"
-        )
+        organisations = Organisation.objects.values("id", "name").order_by("name")
         return json.dumps(list(organisations))
 
-    def get_district_polygons(self):
-        districts = self.districts
-        return json.dumps([district.polygon for district in districts])
-
-    def get_district_names(self):
-        city_wide = _("City wide")
-        districts = AdministrativeDistrict.objects.all()
-        district_names_list = [district.name for district in districts]
-        district_names_list.append(str(city_wide))
-        return json.dumps(district_names_list)
```

Comment:
@goapunk any idea if the district polygons are used anywhere outside the plans, anything to do with bplans? Since @sevfurneaux is removing these methods, would be good to double check with you as well.

Reply:
@sevfurneaux the map on the project list will not be highlighting the district borders any more? If that's the case I think removing this is fine